charts/steward/test/cm_test.go

```diff
@@ -69,7 +69,104 @@ func Test_ConfigPipelineruns(t *testing.T) {
 
 				// VERIFY
 				for key, value := range tc.expectedMapEntries {
-					log.Printf("Key: %s, val: %s, expected: %s", key, cm.Data[key], value)
+					assert.Equal(t, cm.Data[key], value, key)
+				}
+			}
+		})
+	}
+}
+
+func Test_ConfigNetworkPolicies(t *testing.T) {
+	t.Parallel()
+	template := "templates/config-pipelineruns-network-policies.yaml"
+
+	for _, tc := range []struct {
+		name               string
+		values             map[string]string
+		expectedMapEntries map[string]string
+		expectedError      string
+	}{
+		{"empty",
```

Comment:
Need to test for multi-policy with duplicate policies scenario.

Reply:
There cannot be duplicates. See above.